src/openrct2/world/Map.cpp

```diff
@@ -191,14 +191,14 @@ TileElement* map_get_nth_element_at(int32_t x, int32_t y, int32_t n)
     return nullptr;
 }
 
-void map_set_tile_elements(int32_t x, int32_t y, TileElement* elements)
+void map_set_tile_elements(const TileCoordsXY& tilePos, TileElement* elements)
```

Comment:
This is really only setting a single tile element. Perhaps it should be renamed.

Reply:
Yes and no. It sets the pointer to the first element of the tile, which can then be `++`'ed to get to the other tiles. Basically it replaces one chain of elements with another.